bin/pycbc_inspiral

```diff
@@ -197,6 +196,12 @@ parser.add_argument("--checkpoint-exit-maxtime", type=int,
                          " time is exceeded. Default is no checkpointing.")
 parser.add_argument("--checkpoint-exit-code", type=int, default=77,
                     help="Exit code returned if exiting after a checkpoint")
+parser.add_argument("--multiprocessing", type=int,
+                    help="Parallelize over multiple processes, note this is"
+                         "separate from threading using the proc. scheme")
```

Comment:
It's really not clear that this is the number of processes to use? You refer to "nprocesses" in help text below, but that's not what this is called. This reads more like a boolean flag from the help text.

Reply:
I'll rename to be more verbose.